### ISSUE TITLE
Games doesn't run after changing the "Installed Games Directory"

### ISSUE THREAD

**Issue by ZywelZill** [gamejolt/issue-tracker#725]
The situation:
Firstly, I changed the games location to the F drive (F:\games for example), then I installed some games in that path, btw I did not have any game installed before doing this, so, all fine! 

But when I changed the drive letter (from F to Z) and after changing the "Installed Games Directory" to the new path (Z:/games) the client doesn't run any game installed previously.

However, if I install new games with the new path, those games runs normally, the problem is with the already installed games, I guess this issue happens with any change of directory (even in the same drive). if not, is just about external drives.

**Comment by YLivay** [comment]
When you change directory in the client setting it doesn't move the directory of the old games (for obvious reasons, you may have gigabytes and gigabytes of games!) so the client expects to find the games at the same spot you installed them.

This is not a bug but maybe we should consider adding this functionality as a feature.
@hworld?

**Comment by ZywelZill** [comment]
I forgot to mention that I moved all the games to the new directory before changing it from the client (I restarted the client too), what I'm trying to say is that even if the games get moved manually, this still happens.

However, as you say, if I re-install the game, everything is okay because it's installed in the new directory :) for small games that's not a problem, but with too many games and some of them up to GB is a little troublesome.

That's all, just to consider, thanks for replying my post, keep going with the great work, the client is awesome overall! ;)
